app/mailers/application_mailer.rb

```diff
@@ -29,4 +29,8 @@ def uid
   def utm_campaign
     email_event_type
   end
+
+  def general_template
```

Comment:
very minor but general seems an odd word to use? wouldnt default or standard or generic be better word?

Reply:
It's still in WIP. I should have specified in the title. There's so much that breaks around. I had no intention to leave it to `general_template` 🤓 

It will probably just be `template` at the end

Reply:
cool, i didnt see the WIP bit in description tbh!